openfisca_us/parameters/irs/credits/elderly_or_disabled/amount.yaml

```diff
@@ -0,0 +1,15 @@
+description: Initial amount of Section 22 income
+one_qualified:
+  values:
+    2010-01-01: 5_000
```

Comment:
Where is 2010 specified?

Reply:
So far I haven't been able to find a start date - many of the sources on the Cornell Law page give 404s.